protocols.go

```diff
@@ -247,6 +248,11 @@ var (
 		Code:  P_WSS,
 		VCode: CodeToVarint(P_WSS),
 	}
+	protoXWebRTC = Protocol {
```

Comment:
I might be missing something, what's the `X` doing here?
```suggestion
	protoWebRTC = Protocol {
```

Reply:
You also need to add this in `init` (right below).

Reply:
I missed this when removing the transcoders which are now unnecessary. Thank you!